package/yast2-iscsi-client.spec

```
@@ -73,4 +73,5 @@ client.
 %{yast_scrconfdir}/iscsid.scr
 %doc %{yast_docdir}
 %{yast_schemadir}/autoyast/rnc/iscsi-client.rnc
+%{_datadir}/icons/*
```

Please use the `%{yast_icondir}` macro here.

```suggestion
%{yast_icondir}
```